app/models/manageiq/providers/openshift/container_manager/refresh_worker/runner.rb

```diff
@@ -1,2 +1,21 @@
 class ManageIQ::Providers::Openshift::ContainerManager::RefreshWorker::Runner < ManageIQ::Providers::BaseManager::RefreshWorker::Runner
+  include ManageIQ::Providers::Kubernetes::ContainerManager::StreamingRefreshMixin
+
+  def connection_for_entity(entity_type)
+    kubernetes_entity_types.include?(entity_type) ? kubernetes_connection : openshift_connection
+  end
+
+  def openshift_connection
+    @openshift_connection ||= ems.connect
```

Comment:
please specify explicit `connect(:service => "openshift")`.
I want to get rid of openshift manager defaulting to openshift, it voilates Liskov principle.

Reply:
Done